### PR TITLE
Update docs for addition of new property to group object in beta.

### PR DESCRIPTION
**Pull request by samridhigupta**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[37bc7c2](https://github.com/samridhigupta/microsoft-graph-docs/pull/3678/commits/37bc7c2d48e297cf8dad052de0e52a3ffcb31372)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 20 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[37bc7c2](https://github.com/samridhigupta/microsoft-graph-docs/pull/3678/commits/37bc7c2d48e297cf8dad052de0e52a3ffcb31372)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[37bc7c2](https://github.com/samridhigupta/microsoft-graph-docs/pull/3678/commits/37bc7c2d48e297cf8dad052de0e52a3ffcb31372)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/group.md](https://github.com/samridhigupta/microsoft-graph-docs/blob/master/api-reference/beta/resources/group.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/samridhigupta/microsoft-graph-docs/blob/master/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C20%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812201948345920-3678%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=R1%2B%2BMMR7voez%2Bn5lgSg3BBDYH3Ea5oARWjvOkCPklzs%3D&st=2018-12-20T19%3A47%3A00Z&se=2019-01-20T19%3A52%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Lauragra** [comment]
@lleonard-msft , please approve this PR and set the ready to merge label if it's ready to go. Thanks.

**Comment by lleonard-msft** [comment]
@dkershaw10 - Please review and sign-off

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a573018](https://github.com/samridhigupta/microsoft-graph-docs/pull/3678/commits/a573018deb8e5374370766186c6b49f3803a3ef1)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 21 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a573018](https://github.com/samridhigupta/microsoft-graph-docs/pull/3678/commits/a573018deb8e5374370766186c6b49f3803a3ef1)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by Lauragra** [comment]
@Dan Kershaw<mailto:dkershaw@microsoft.com>, do you need to review/sign off on updates to group properties?

Thanks,
Laura

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a573018](https://github.com/samridhigupta/microsoft-graph-docs/pull/3678/commits/a573018deb8e5374370766186c6b49f3803a3ef1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/group.md](https://github.com/samridhigupta/microsoft-graph-docs/blob/master/api-reference/beta/resources/group.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/samridhigupta/microsoft-graph-docs/blob/master/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C9%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901090200450509-3678%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=qLzx%2F7pCITPSxshocjdeiR0Kn0O8q9vUJ8Td5Hqxtkw%3D&st=2019-01-09T01%3A59%3A45Z&se=2019-02-09T02%3A04%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/resources/group.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ This resource supports:
 |createdDateTime|DateTimeOffset| Timestamp of when the group was created. The value cannot be modified and is automatically populated when the group is created. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`. Read-only. |
 |description|String|An optional description for the group.|
 |displayName|String|The display name for the group. This property is required when a group is created and it cannot be cleared during updates. Supports $filter and $orderby.|
+|expirationDateTime|DateTimeOffset| Timestamp of when the group will expire. The value cannot be modified and is populated based on expiration policy. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`. Read-only. |
```
Comment (Lauragra) [inline api-reference/beta/resources/group.md]: Please indicate whether this property is returned by default. For guidance, see "Default properties" in the Microsoft Graph CDK.

**Comment by samridhigupta** [comment]
Yuan already made the change to the docs, this PR is not needed any more.

**Pull request closed without merging**